src/utils/validators.js

```diff
@@ -317,3 +319,46 @@ export const isTimePointSec = (v) => {
 		return false;
 	}
 };
+
+export const isAccountIdOrName = (v) => {
+	if (!isAccountId(v) && !isAccountName(v)) throw new Error('Accounts id or name should be string and valid');
+};
+
+export const isAssetIdOrName = (v) => {
+	if (!isAssetId(v) && !isAssetName(v)) throw new Error('Assets id or name should be string and valid');
+};
+
+export const isMethodExists = (v) => {
+	if (!walletAPIMethodsArray.includes(v)) throw new Error('This method does not exists');
+};
+
+export const isOperationPrototypeExists = (v) => {
+	if (!operationPrototypeArray.includes(v)) throw new Error('This operation does not exists');
+};
+
+export const isNotEmptyString = (v) => isString(v) && !!v.trim();
+
+export const isContractCode = (v) => v === '' || (isHex(v) && v.length % 2 === 0);
+
+
+export const isOldPrivateKey = (v) => {
+	if (!isString(v)) return false;
+
+	if (bs58.decode(v).length !== LENGTH_DECODE_PRIVATE_KEY) {
+		throw new Error('Invalid private key');
+	}
+
+	return true;
+};
+
+const validateAmount = (v) => {
+	if (!isString(v)) return false;
+
+	const integer = Math.abs(Number(v.split('.')[0]));
+
+	if (integer > (AMOUNT_MAX_NUMBER)) return false;
+
+	return (integer * 1e12 < ECHO_MAX_SHARE_SUPPLY);
+};
```

Comment:
Methods like `isSomething(v)` should not throws errors, but just returns boolean